tools/src/main/java/org/apache/kafka/tools/automq/AutoMQAdminCmd.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.tools.automq;
+
+import net.sourceforge.argparse4j.ArgumentParsers;
+import net.sourceforge.argparse4j.inf.ArgumentParser;
+
+import static net.sourceforge.argparse4j.impl.Arguments.store;
+
+public class AutoMQAdminCmd {
+
+    public static final String ADMIN_TOOL_NAME = "automq-kafka-admin.sh";
+
+    public static final String GENERATE_S3_URL_CMD = "generate-s3-url";
+    public static final String GENERATE_START_COMMAND_CMD = "generate-start-command";
+    public static final String GENERATE_CONFIG_PROPERTIES_CMD = "generate-config-properties";
```

Comment:
I think it's better to set these three commands as sub-commands, see `argparse4j.inf.Subparser`.